ffwd-client/pom.xml

```diff
@@ -14,10 +14,24 @@
     <description>FastForward client for Java.</description>
 
     <dependencies>
+        <dependency>
+            <groupId>com.google.auto.value</groupId>
+            <artifactId>auto-value</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+            <version>1.3.2</version>
+        </dependency>
         <dependency>
             <groupId>com.google.protobuf</groupId>
             <artifactId>protobuf-java</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
```

Comment:
Could introduction of guava dependency here create some issues downstream? I don't for sure if this could be an issue but all those java dependency hell issues.